### PR TITLE
adding fetch_metrics.py to execute fio_metrics and vm_metrics

### PR DESCRIPTION
**Pull request by adata111**

### CODE REVIEW

File: perfmetrics/scripts/fetch_metrics.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  argv = sys.argv
+  if len(argv) != 2:
+    raise TypeError('Incorrect number of arguments.\n'
+                    'Usage: '
+                    'python3 fetch_metrics.py <fio output json filepath>')
+
+  fio_metrics_obj = fio_metrics.FioMetrics()
+  print('Getting fio metrics...')
+  temp = fio_metrics_obj.get_metrics(argv[1])
+  print('Waiting for 250 seconds for metrics to be updated on VM...')
```
Comment (vadlakondaswetha) [inline perfmetrics/scripts/fetch_metrics.py:24]: add a comment here why you are waiting for 250 sec

Comment (vadlakondaswetha) [inline perfmetrics/scripts/fetch_metrics.py:24]: can we add a new line here to seperate out fio metrics logic and vm metrics logic

**Review by vadlakondaswetha — APPROVED** [review]

File: perfmetrics/scripts/fetch_metrics.py (hunk 1 above)
Comment (adata111) [reply perfmetrics/scripts/fetch_metrics.py:24]: According to the Google Cloud Monitoring docs, it takes up to 240 seconds for sampled data to be visible on the VM metrics graph. Therefore, I am waiting for 250 seconds to make sure the returned metrics are not empty.
Doc link: [https://cloud.google.com/monitoring/api/metrics_gcp#:~:text=Count%20of%20bytes%20received%20from%20the%20network.%20Sampled%20every%2060%20seconds.%20After%20sampling%2C%20data%20is%20not%20visible%20for%20up%20to%20240%20seconds.](url)

Comment (adata111) [reply perfmetrics/scripts/fetch_metrics.py:25]: done

**Review by vadlakondaswetha — APPROVED** [review]

**Pull request merged**